openmls/src/group/core_group/new_from_external_init.rs

```diff
@@ -136,7 +88,7 @@ impl CoreGroup {
             index,
             signature_key,
             ..
-        } in treesync.full_leave_members()
+        } in public_group.treesync().full_leave_members()
```

Comment:
Shall we properly define the `Member` as part of the public as well and make use of it in publi APIs (like remove)? Not in this PR but maybe in a follow up?